test-utils/src/data/free.rs

```diff
@@ -6,105 +6,260 @@
+        tx_blobs: [$($tx_blob:ident),*], // Array of contained transaction blobs
+        pow_hash: $pow_hash:literal, // PoW hash as a string literal
+        height: $height:literal, // Block height
+        generated_coins: $generated_coins:literal, // Generated coins in block (`reward`)
+        weight: $weight:literal, // Block weight
+        long_term_weight: $long_term_weight:literal, // Block long term weight
+        cumulative_difficulty: $cumulative_difficulty:literal, // Block cumulative difficulty
+        tx_len: $tx_len:literal, // Amount of transactions in this block
+    ) => {
+        #[doc = concat!(
+            "Return [`",
+            stringify!($block_blob),
+            "`] as a [`VerifiedBlockInformation`].",
+        )]
+        ///
+        /// Contained transactions:
+        $(
+            #[doc = concat!("- [`", stringify!($tx_blob), "`]")]
+        )*
+        ///
+        /// ```rust
+        #[doc = "# use cuprate_test_utils::data::*;"]
+        #[doc = "# use hex_literal::hex;"]
+        #[doc = concat!("let block = ", stringify!($fn_name), "();")]
+        #[doc = concat!("assert_eq!(&block.block.serialize(), ", stringify!($block_blob), ");")]
+        #[doc = concat!("assert_eq!(block.pow_hash, hex!(\"", $pow_hash, "\"));")]
+        #[doc = concat!("assert_eq!(block.height, ", $height, ");")]
+        #[doc = concat!("assert_eq!(block.generated_coins, ", $generated_coins, ");")]
+        #[doc = concat!("assert_eq!(block.weight, ", $weight, ");")]
+        #[doc = concat!("assert_eq!(block.long_term_weight, ", $long_term_weight, ");")]
+        #[doc = concat!("assert_eq!(block.cumulative_difficulty, ", $cumulative_difficulty, ");")]
+        #[doc = concat!("assert_eq!(block.txs.len(), ", $tx_len, ");")]
+        /// ```
+        pub fn $fn_name() -> &'static VerifiedBlockInformation {
+            static BLOCK: OnceLock<VerifiedBlockInformation> = OnceLock::new();
+            BLOCK.get_or_init(|| {
+                VerifiedBlockMap {
+                    block: Block::read(&mut $block_blob).unwrap(),
+                    pow_hash: hex!($pow_hash),
+                    height: $height,
+                    generated_coins: $generated_coins,
+                    weight: $weight,
+                    long_term_weight: $long_term_weight,
+                    cumulative_difficulty: $cumulative_difficulty,
+                    txs: vec![$($tx_blob),*],
+                }
+                .into_verified()
+            })
+        }
+    };
+}
+
+verified_block_information_fn! {
+    fn_name: block_v1_tx2,
+    block_blob: BLOCK_5ECB7E,
+    tx_blobs: [TX_2180A8, TX_D7FEBD],
+    pow_hash: "84f64766475d51837ac9efbef1926486e58563c95a19fef4aec3254f03000000",
+    height: 202_612,
+    generated_coins: 13_138_270_468_431,
+    weight: 55_503,
+    long_term_weight: 55_503,
+    cumulative_difficulty: 126_654_460_829_362,
+    tx_len: 2,
+}
```

Comment:
Less-than-ideal part for defining blocks is that we have to provide each TX blob, so I swapped out the previous one that had 513.